pkg/rbac/sharedData_test.go

```diff
@@ -96,10 +96,11 @@ func Test_getResouces_usingCache(t *testing.T) {
 		gomock.Eq([]interface{}{}),
 	).Return(pgxRows, nil)
 
-	var managedCluster, namespaces []string
+	var namespaces []string
 
 	namespaces = append(namespaces, "test-namespace")
-	managedCluster = append(managedCluster, "test-man")
+	managedCluster := make(map[string]struct{})
+	managedCluster["test-man"] = struct{}{}
```

Comment:
Suggestion: Could compress this to a single line. (Several instances in this file)
```suggestion
	manClusters := map[string]struct{}{"test-man": struct{}{}}
```

Reply:
Updated